Tasks/Common/codeanalysis-common/package-lock.json

```diff
@@ -51,7 +51,7 @@
       "integrity": "sha1-Y7xdy2EzG5K8Bf1SiVPDNGKgb40=",
       "optional": true,
       "requires": {
-        "tweetnacl": "0.14.5"
+        "tweetnacl": "^0.14.3"
```

Comment:
Not sure why your PR should change the package lock for these other common packages.  This one for instance is odd, loosens the dependency version.